datafusion/src/physical_plan/functions.rs

```diff
@@ -1358,6 +1366,18 @@ impl fmt::Display for ScalarFunctionExpr {
     }
 }
 
+/// null columnar values are implemented as a null array in order to pass batch
+/// num_rows
+type NullColumnarValue = ColumnarValue;
+
+impl TryFrom<&RecordBatch> for NullColumnarValue {
+    type Error = DataFusionError;
+    fn try_from(batch: &RecordBatch) -> Result<Self> {
```

Comment:
It seems to me this function is infallible (aka it never returns error, but always returns `Ok`).

Perhaps we could implement `From` instead of `TryFrom`?

Reply:
updated